### ISSUE TITLE
Check what we can do for Symfony flex

### ISSUE THREAD

**Issue by Haehnchen** [Haehnchen/idea-php-symfony2-plugin#922]
- composer packages alias: https://symfony.sh/aliases.json

**Comment by King2500** [comment]
Rename existing "Symfony" project template to "Symfony Standard Edition" and add "Symfony Flex" with selection of recipes ?

check required PHP version 7.1.

project based on symfony/skeleton:
`composer create-project symfony/skeleton:3.3.x-dev demo`

recipes:
https://github.com/symfony/recipes
https://github.com/symfony/recipes-contrib

Symfony 4 demo:
https://medium.com/@fabpot/symfony-4-a-quick-demo-da7d32be323

**Comment by gharlan** [comment]
A tip for flex users:

Add these two lines manually:

![screenshot 2017-08-02 15 50 53](https://user-images.githubusercontent.com/330436/28876715-64f9381e-779a-11e7-9693-2d13f5ea2b3f.png)

After that jumping between template and controller is working again. :)

But I couldn't find a workaround for goto-translation.

**Comment by Koc** [comment]
@Haehnchen does Symfony Flex projects properly detected and PS suggest to enable Symfony plugin?

**Comment by Haehnchen** [comment]
yes, in additional every single component will trigger it via: https://github.com/Haehnchen/idea-php-symfony2-plugin/commit/f1779090c3f118785d8f479a384359dc196c03b7

**Comment by King2500** [comment]
I suggest checking the requirements in composer.json. The user might not directly use Symfony components but other libraries might require them (Symfony console is very wide-spread). So people might be confused why the Symfony plugin got enabled without them "using" Symfony components.

**Comment by Haehnchen** [comment]
> Symfony console is very wide-spread

Thats the point. this plugin provides not only support for a Symfony fullstack application. there is for example also a console arguments completion. so we are moving away from this fullstacks

**Comment by King2500** [comment]
Do you have an example of a third-party library with uses Symfony console and you profit from the plugin's features (indirectly)?
